doc/releases.rst

```diff
@@ -11,6 +11,31 @@
 Releases
 ========
 
+Version 0.7.1
```

Comment:
Hey, are you maybe missing a fix for #715 here? Could you please also already include your PR for #884 here so that we don't have to update the release notes again?